packages/jest-config/src/read_config_file_and_set_root_dir.js

```diff
@@ -51,8 +51,8 @@ export default (configPath: Path): InitialOptions => {
       );
     }
   } else {
-    // If rootDir is not there, we'll set it to this file's __dirname
-    configObject.rootDir = path.dirname(configPath);
+    // If rootDir is not there, we'll set it to cwd
+    configObject.rootDir = process.cwd();
```

Comment:
I don't think this is the correct change - from what I can gather it looks at the filename now to support `projects`.

I'm not really sure what the correct behavior should be. I'm leading towards using `cwd` and requiring `rootDir` in the config of nested `projects`.